src/AutorestSwift/JsonRpc/AutorestPlugin.swift

```diff
@@ -36,8 +36,7 @@ class AutorestPlugin {
     var processRequestId: Int!
 
     // TODO: Increase when working correctly
-    private let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
-    private let group = DispatchGroup()
+    var eventLoop = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
Why do we need need more than 1 thread? even when working correctly?

Reply:
I haven't tried changing this to see, but it could speed things up a bit when blocking I/O is happening like writing files.